app/src/main/java/agency/five/codebase/android/movieapp/ui/home/mapper/HomeScreenMapperImpl.kt

```diff
@@ -0,0 +1,43 @@
+package agency.five.codebase.android.movieapp.ui.home.mapper
+
+import agency.five.codebase.android.movieapp.R
+import agency.five.codebase.android.movieapp.model.Movie
+import agency.five.codebase.android.movieapp.model.MovieCategory
+import agency.five.codebase.android.movieapp.model.MovieCategory.*
+import agency.five.codebase.android.movieapp.ui.component.MovieCategoryLabelTextViewState
+import agency.five.codebase.android.movieapp.ui.component.MovieCategoryLabelViewState
+import agency.five.codebase.android.movieapp.ui.home.HomeMovieCategoryViewState
+import agency.five.codebase.android.movieapp.ui.home.HomeMovieViewState
+
+class HomeScreenMapperImpl : HomeScreenMapper {
+
+    override fun toHomeMovieCategoryViewState(
+        movieCategories: List<MovieCategory>,
+        selectedMovieCategory: MovieCategory,
+        movies: List<Movie>
+    ): HomeMovieCategoryViewState {
+        return HomeMovieCategoryViewState(
```

Comment:
make this expression function instead of function with body

Reply:
https://stackoverflow.com/questions/64856717/kotlin-expression-fun-vs-normal-fun-differences